tests/myrapname.test.js

```diff
@@ -1,6 +1,84 @@
+const { chromium } = require('playwright')
+
 describe("Myrapname.com", () => {
+  let browser
+  let page
+
+  beforeAll(async () => {
+    browser = await chromium.launch({headless: false, slowMo: 50})
```

Comment:
I put in the slowmo in order to see what the test was doing a bit clearer. This can be removed now